packages/cerebral/src/Controller.js

```diff
@@ -16,9 +17,11 @@ import {dependencyStore as computedDependencyStore} from './Computed'
   based on top level providers and providers defined in modules
 */
 class Controller extends EventEmitter {
-  constructor ({state = {}, signals = {}, providers = [], modules = {}, router, devtools = null, strictRender = false}) {
+  constructor ({state = {}, signals = {}, providers = [], modules = {}, router, devtools = null, options = {}}) {
     super()
-    this.strictRender = strictRender
+    this.computedDependencyStore = computedDependencyStore
```

Comment:
Hm, issue here is that computed does not know anything about controllers. There is no "create instance of computed"... cause we do: `import {Computed} from 'cerebral'`.

We would have to change this to: `import {Computed} from './controller'`

Sharing computed store across controllers is no problem though. You might get into an extreme edge case where one controller flags the computed used in a different controller... cause they for some reason have the exact same state path. But this is not a real issue.

So suggest that we keep "as is"... cause this is not a real issue and very likely will never be ;-)